x-pack/plugins/ml/server/models/bucket_span_estimator/bucket_span_estimator.js

```diff
@@ -343,11 +343,21 @@ export function estimateBucketSpanFactory(
           filterPath: '*.*max_buckets',
         })
           .then(settings => {
-            if (typeof settings !== 'object' || typeof settings.defaults !== 'object') {
+            if (typeof settings !== 'object') {
+              reject('Unable to retrieve cluster settings');
```

Comment:
shall we add i18n for this message?

Reply:
we generally don't translate errors thrown from the server side, none of the other errors thrown by the bucketspan estimator are translated nor are errors in other services like the data recogniser.
This might be an oversight and we should be translating all of these, if so i would suggest we do that in a follow up PR.